app/src/test/java/de/westnordost/streetcomplete/quests/barrier_type/AddStileTypeTest.kt

```diff
@@ -31,6 +31,7 @@ class AddStileTypeTest {
             ),
             StileType.SQUEEZER,
             StringMapEntryAdd("check_date", LocalDate.now().toCheckDateString()),
+            StringMapEntryModify("stile", "squeezer", "squeezer"),
```

Comment:
I meant to flag this with the previous changes, but should this really generate a modify when nothing has actually changed? It should be pretty easy to fix and seems like it would be a good thing to me...

Reply:
Yes, this is deliberate. The user answered in this case that there is **still a squeezer**. Now imagine the element changed since then - someone changed `stile=stepover` or whatever. So the change is checked for conflicts. If this entry wouldn't be in the edit made by the user, StreetComplete would just add the check date and be done with it, while it would be preferrable that the app detected a conflict here.